lapack/testlapack/general.go

```diff
@@ -1450,42 +1450,42 @@ func dlange(norm lapack.MatrixNorm, m, n int, a []float64, lda int) float64 {
 	if m == 0 || n == 0 {
 		return 0
 	}
+	var value float64
 	switch norm {
 	case lapack.MaxAbs:
-		var value float64
 		for i := 0; i < m; i++ {
 			for j := 0; j < n; j++ {
 				value = math.Max(value, math.Abs(a[i*lda+j]))
 			}
 		}
-		return value
 	case lapack.MaxColumnSum:
 		work := make([]float64, n)
 		for i := 0; i < m; i++ {
 			for j := 0; j < n; j++ {
 				work[j] += math.Abs(a[i*lda+j])
 			}
 		}
-		var value float64
 		for i := 0; i < n; i++ {
 			value = math.Max(value, work[i])
 		}
-		return value
 	case lapack.MaxRowSum:
-		var value float64
 		for i := 0; i < m; i++ {
 			var sum float64
 			for j := 0; j < n; j++ {
 				sum += math.Abs(a[i*lda+j])
 			}
 			value = math.Max(value, sum)
 		}
-		return value
 	case lapack.Frobenius:
-		panic("not implemented")
+		for i := 0; i < m; i++ {
+			for j := 0; j < n; j++ {
+				value = math.Hypot(value, a[i*lda+j])
+			}
+		}
```

Comment:
We have `L2NormInc` in ".../internal/asm/f64", but that won't work here unless `m` is the stride. That code does more work to avoid over an underflow. Do we want that here? Probably not given that this is testing code.

Reply:
Thanks for reminding me about `L2NormInc`. It can be used here to compute the row norms and then put them together with `math.Hypot`. For testing code I think this is enough. PTAL